src/System.IO.Compression/tests/Performance/Perf.DeflateStream.cs

```diff
@@ -97,22 +97,24 @@ public enum CompressionType
             NormalData = 4
         }
 
-        [Benchmark]
+        private const int Iter = 1000;
+
+        [Benchmark(InnerIterationCount = Iter)]
```

Comment:
This is reducing the number of inner iterations from 10000 to 1000. Is that on purpose, and if so, why?